devel/datree/Portfile

```
@@ -0,0 +1,414 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/datreeio/datree 1.6.26
+
+categories          devel
+maintainers         {@Ci7rix unset.ch:max+github} openmaintainer
+license             Apache-2
+
+description         CLI tool to run policies against Kubernetes manifests YAML files or Helm charts
+long_description    Datree automatically validates Kubernetes objects for rule violations, ensuring no misconfigurations reach production
+
+checksums           ${distname}${extract.suffix} \
+                    rmd160  cbe94d55f7651a32031983d0e6d6dd8d999e0554 \
+                    sha256  a31dd4a3d304c13ea0a0c40fa6f53614a61c708894dba709251ca135ac553f03 \
+                    size    5444221
+
+set go_ldflags      "-s -w -X github.com/datreeio/datree/cmd.CliVersion=${version}"
```

What you can do here is use `${go.package}` to avoid repetition:

```suggestion
set go_ldflags      "-s -w -X ${go.package}/cmd.CliVersion=${version}"
```

---

Done ! Thank you for your feedback 👍